web/README.md

```diff
@@ -1,46 +1,163 @@
+<h1 align="center">
```

Comment:
You should not copy the root README in the web folder

Use a README.md file that is dedicated to web or remove it completely